README.md

```diff
@@ -36,6 +36,13 @@ Add Firebase to your project if you haven't already ([Android](https://firebase.
 
 See [Add Firebase Crashlytics to your app](https://firebase.google.com/docs/crashlytics/get-started?platform=android#add-sdk) and follow the instructions to set up your app correctly.
 
+### iOS
+
+See [Add Firebase Crashlytics to your app](https://firebase.google.com/docs/crashlytics/get-started?platform=ios#add-sdk) and follow the instructions to set up your app correctly.
+
+Important: Do not attach a debugger at launch. It may happen that the dSYM files are not uploaded automatically. However, this does not prevent the crash from being displayed in Crashlytics.
```

Comment:
> Important: Do not attach a debugger at launch.

This is already mentioned in the linked [Firebase documentation](https://firebase.google.com/docs/crashlytics/force-a-crash?platform=ios). I prefer to get such information directly from the official documentation, as it is always up to date.

> It may happen that the dSYM files are not uploaded automatically. However, this does not prevent the crash from being displayed in Crashlytics.

This should be fixed in the current version (`v4.0.0`). I only mentioned this because I didn't know what plugin version you were using.